content/10.js/01.web3.js/01.getting-started.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Getting Started
+description: How to install and initialize the Web3.js plugin for ZKsync
+tags: ["zksync", "sdks", "zksync era sdk", "ethereum", "era network", "web3.js", "web3.js plugin", "installation", "setup"]
+---
+
+Follow these steps to get started with the Web3.js plugin for ZKsync.
+
+## Installation
+
+Use a package manager like npm to install Web3.js and the Web3.js plugin for ZKsync.
+
+```sh
+npm install web3 web3-plugin-zksync
+```
+
+## Register the Plugin
+
+To use the Web3.js plugin for ZKsync, register it with a Web3 instance.
+
+```ts
+import { Web3 } from "web3";
+import { ZKsyncPlugin } from "web3-plugin-zksync";
+
+async function main() {
+  const web3: Web3 = new Web3("http://localhost:8545/");
+  web3.registerPlugin(new ZKsyncPlugin("http://localhost:3050/"));
```

Comment:
Can we use testnet URLs here for the example, along with comments defining them?

Also, which version of web3-plugin-zksync can I test these scripts? I am using alpha and typescript is having issues with the plugin module definition.

Reply:
Yes, I can use testnet URLs if that is what you prefer.  For the L2 testnet URL I will use `https://sepolia.era.zksync.dev`. What testnet URL would you like me to use for the L1 provider?

I think it's expected that there will be some problems with the code snippets in these docs because the latest version of the plugin has not been published.

Reply:
When will the latest version of the plugin be published, or is it one of the alpha versions?

Reply:
> What testnet URL would you like me to use for the L1 provider?

Does an L1 provider have to be added to use with ZKsync?

Reply:
> When will the latest version of the plugin be published, or is it one of the alpha versions?

The latest version will be published later this week (I don't believe an alpha version has been published). cc @avkos @Muhammad-Altabba

> Does an L1 provider have to be added to use with ZKsync?

I do not know. Maybe @avkos @Muhammad-Altabba can also comment on that 🙏🏻

Reply:
@itsacoyote - The plugin has been published and the problem you were experiencing with the plugin module definition should be fixed. I have updated the documentation to use testnet URLs for the examples and removed the L1 provider completely (I confirmed with the development team that all L2 functionality will work as intended when the plugin is registered with a Web3.js instance that is not connected to a provider). I'm leaving this PR in a draft state, though, because there is [an issue with the API documentation](https://github.com/ChainSafe/web3-plugin-zksync/actions/runs/10512177177/job/29124863191) so there are currently broken links.

Reply:
Thank you, @avkos 🙏🏻 @itsacoyote I have moved this PR out of the draft state ✅